lib/npg_pipeline/pluggable/registry.pm

```diff
@@ -48,7 +48,9 @@ Readonly::Hash my %REGISTRY => (
 
   'bam_cluster_counter_check'=> {'cluster_count' => 'create'},
   'seqchksum_comparator'     => {'seqchksum_comparator' => 'create'},
-                               );
+  'archive_to_s3'            => {'s3_archiver' => 'create'},
```

Comment:
Do we want to add the new functions to data/config_files/function_list_post_qc_review.json? Or are we goign to run them manually to start with?

Reply:
Best if they go straight into the post_qc_review pipeline (but that can be in either this PR or one soon after) - I'll be using this PR to test archival...